Quadriga/src/main/java/edu/asu/spring/quadriga/service/impl/publicwebsite/AboutTextManager.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.spring.quadriga.service.impl.publicwebsite;
+
+import org.hibernate.HibernateException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import edu.asu.spring.quadriga.dao.publicwebsite.IAboutTextDAO;
+import edu.asu.spring.quadriga.domain.settings.IAboutText;
+import edu.asu.spring.quadriga.dto.AboutTextDTO;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.publicwebsite.IAboutTextManager;
+
+/**
+ * Service to save title and description of public website about page form.
+ * 
+ * @author Rajat Aggarwal
+ *
+ */
+@Service
+public class AboutTextManager implements IAboutTextManager {
+
+    @Autowired
+    private IAboutTextDAO aboutTextDAO;
+
+    @Transactional
+    @Override
+    public void saveAbout(String projectId, String title, String description) throws QuadrigaStorageException {
+
+        AboutTextDTO aboutTextDTO = aboutTextDAO.getDTOByProjectId(projectId);
+
+        if (aboutTextDTO == null) {
+            aboutTextDTO = new AboutTextDTO();
+            aboutTextDTO.setId(aboutTextDAO.generateUniqueID());
+            aboutTextDTO.setProjectId(projectId);
+            aboutTextDTO.setTitle(title);
+            aboutTextDTO.setDescription(description);
+            aboutTextDAO.saveNewDTO(aboutTextDTO);
+        } else {
+            aboutTextDTO.setProjectId(projectId);
+            aboutTextDTO.setTitle(title);
+            aboutTextDTO.setDescription(description);
```

Comment:
Don't repeat code. you can simply create a new AboutTextDTO when null is returned from the dao. Then set project id, title, and description and in the do an if on the id. if it's set update, if it's null create one and save.